app/shared/components/Wallet/Panel/Button/Stake.js

```diff
@@ -85,6 +85,7 @@ export default class WalletPanelButtonStake extends Component<Props> {
                   validate={validate}
                   balance={balances[settings.account]}
                   system={system}
+                  t={t}
```

Comment:
The stake form itself should be wrapped in a `translate` call as opposed to passing the `t` method.

Reply:
But don't you also need to pass the t method? I thought wrapping everything in a translate method was used only when changing the locale scope.

Reply:
Just include and wrap the new element with `translate`.

I am assuming that wrapping a new element in it's own translate instance is a better practice than being forced to pass it constantly.